bundler/spec/other/ext_spec.rb

```diff
@@ -1,22 +1,24 @@
 # frozen_string_literal: true
 
-RSpec.describe "Gem::Specification#match_platform" do
+RSpec.describe "Gem::Specification#installable_on_platform?" do
   it "does not match platforms other than the gem platform" do
     darwin = gem "lol", "1.0", "platform_specific-1.0-x86-darwin-10"
-    expect(darwin.match_platform(pl("java"))).to eq(false)
+    expect(darwin.installable_on_platform?(pl("java"))).to eq(false)
   end
 
   context "when platform is a string" do
     it "matches when platform is a string" do
       lazy_spec = Bundler::LazySpecification.new("lol", "1.0", "universal-mingw32")
-      expect(lazy_spec.match_platform(pl("x86-mingw32"))).to eq(true)
-      expect(lazy_spec.match_platform(pl("x64-mingw32"))).to eq(true)
+      expect(lazy_spec.installable_on_platform?(pl("x86-mingw32"))).to eq(true)
+      expect(lazy_spec.installable_on_platform?(pl("x64-mingw32"))).to eq(true)
     end
   end
 end
 
 RSpec.describe "Bundler::GemHelpers#generic" do
```

Comment:
This module is now gone, so this whole block should be removed and merged with `Gem::Platform.generic` specs in RubyGems.